### DISCUSSION TITLE
The "make" command infinitely recurses, forever re-creating Makefile, and the product is never built

### DISCUSSION THREAD

**Discussion by shachter** [starg2/timidity41#3] [Q&A]
When I invoke the "configure" command (parenthetically, why is it not executable?  I unzip the archive with a umask of 0, and configure is created without execute permissions), it creates an unusable Makefile: when I subsequently invoke the "make" command, the first thing it does is invoke "/bin/sh config.status", which replays the configure command, with all its arguments, which recreates all the Makefiles, and this proceeds in an infinite loop.  I am using a standard Linux distribution with no idiosyncratic customizations.  Has anyone ever attempted to actually build this product?  If so, how did you do it?

     Jay F. Shachter
     jay at m5 dot chicago dot il dot us

**Comment by starg2** [comment]
Support for Linux and autotools is totally broken now. This project diverged significantly from the original one, but most people involved in development only cared about Windows. The only supported way to build this project now is to use CMake and Ninja on Windows. PRs are welcome to restore support.
